{
  "can_take_orders": false,
  "reasons": [
    {
      "code": "DriverTooManyChains",
      "message": "Driver already has chain order in reserve, ids=id0"
    }
  ]
}
